script.js

```diff
@@ -28,19 +28,70 @@ randomQuote();
 // Part 1
 
 
+ var mainTitle = document.querySelector('#main-title').textContent.replace('Hi! My name is DOM Toretto.');
```

Comment:
no need for replace
`var mainTitle = document.querySelector('#main-title').textContent = 'Hi! My name is DOM Toretto.'